chromeExtension/package.json

```diff
@@ -1,45 +1,46 @@
-{
-  "name": "gotdibbs-toolbox",
-  "version": "2.3.3",
-  "description": "Admin tools for Dynamics 365/CRM",
-  "main": "index.js",
-  "scripts": {
-    "dev": "webpack --progress --mode development --env NODE_ENV='development' --config ./webpack/webpack.config.js",
-    "prod": "webpack --progress --mode production --env NODE_ENV='production' --config ./webpack/webpack.config.js",
-    "test:integration": "cd tests/integration/ && npm run test && cd ../../"
-  },
-  "author": "",
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/core": "^7.11.0",
-    "@babel/preset-env": "^7.11.0",
-    "@babel/preset-react": "^7.10.4",
-    "babel-loader": "^8.1.0",
-    "classnames": "^2.2.6",
-    "copy-webpack-plugin": "^7.0.0",
-    "core-js": "^3.8.2",
-    "crx": "^5.0.1",
-    "css-loader": "^5.0.1",
-    "dotenv": "^8.2.0",
-    "eslint": "^7.17.0",
-    "eslint-config-defaults": "^9.0.0",
-    "eslint-loader": "^4.0.2",
-    "eslint-plugin-babel": "^5.3.1",
-    "eslint-plugin-react": "^7.20.5",
-    "extract-css-chunks-webpack-plugin": "^4.7.5",
-    "react-hook-form": "^6.14.1",
-    "react-json-view": "^1.19.1",
-    "style-loader": "^2.0.0",
-    "sumo-logger": "^2.6.1",
-    "url-loader": "^4.1.0",
-    "webpack": "^5.14.0",
-    "webpack-cli": "^4.3.1",
-    "zip-webpack-plugin": "^4.0.1"
-  },
-  "dependencies": {
-    "@honeybadger-io/react": "^0.0.8",
-    "fathom-client": "^3.0.0",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
-  }
-}
+{
+  "name": "gotdibbs-toolbox",
+  "version": "2022.1",

```

Comment:
Changed to new versioning scheme to help make it more clear when there's been an update, and as a kick in my pants to keep things updated.